platform/win32.js

```diff
@@ -27,9 +27,8 @@ class SayPlatformWin32 extends SayPlatformBase {
       psCommand += `$speak.Rate = ${adjustedSpeed};`
     }
 
-    psCommand += `$speak.Speak([Console]::In.ReadToEnd())`
-
-    pipedData += text
+    psCommand += `$speak.Speak('${text}')`
+    
```

Comment:
Can you delete this line? Looks like it's failing the linter check.

Reply:
I got the chinese support working by removing `'` from the text when passing to Speak().

```suggestion
psCommand += `$speak.Speak('${text.replace(/'/g, "")}')`;
```

Test:

```js
say.getInstalledVoices(function(err, names){
    console.log( "%j", arguments ); 
    const voice = names[names.length- 1];
    console.log("using voice: %j", voice);
    
    say.speak("Hi. What's up dude? 我记下来了", voice, 1.0, (err) => {
        if (err) {
          return console.error(err)
        }      
        console.log('Text has been spoken.')
      });      
});
```